src/components/WeatherWidget/WeatherWidget.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useEffect } from 'react'
+import { Button } from '@trussworks/react-uswds'
+import styles from '../NewsWidget/NewsWidget.module.scss'
+import { WidgetWithSettings } from 'components/Widget/Widget'
+import { useWeather } from 'hooks/useWeather'
+import { useModalContext } from 'stores/modalContext'
+import { WeatherWidget as WeatherWidgetType } from 'graphql.g'
+import { WidgetType } from 'graphql.g'
+import { useEditWeatherWidgetMutation } from 'operations/portal/mutations/editWeatherWidget.g'
+
+// #TODO This is a starter widget for WeatherWidget
```

Comment:
This is for demo/testing purposes to show @jcbcapps the getForecast query, and edit+remove functionality for the Weather Widget. I can either remove from this PR once he's branched off, or we can leave it as a WIP since it won't be rendered anywhere.